packages/config/config/devices/0x0212/gda-a2e6b-k0.json

```diff
@@ -0,0 +1,88 @@
+// GATEMAN GDA-A2E6B-K0
+// A20-SH
+{
+	"manufacturer": "GATEMAN",
+	"manufacturerId": "0x0212",
+	"label": "GDA-A2E6B-K0",
+	"description": "A20-SH",
+	"devices": [
+		{
+			"productType": "0x1000",
+			"productId": "0x1000",
+			"zwaveAllianceId": 1434
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Silent Mode On/Off",
+			"description": "",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 3,
+			"defaultValue": 3,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Mute",
+					"value": 1
+				},
+				{
+					"label": "Volume low",
+					"value": 2
+				},
+				{
+					"label": "Volume high",
+					"value": 3
+				}
+			]
+		},
+		"2": {
+			"label": "Auto Relock On/Off",
+			"description": "Set the auto relock on/off",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable auto relock",
+					"value": 0
+				},
+				{
+					"label": "Enable auto relock",
+					"value": 255
+				}
+			]
+		},
+		"3": {
+			"label": "Relock Time",
+			"description": "This value from 10 to 180 seconds means the time for lock to retain the unlocked state before doing the auto relock",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 10,
+			"maxValue": 180,
+			"defaultValue": 30,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		}
+	},
+	"metadata": {
+		"inclusion": "① Touch Keypad with palm.\n② Enter User Code(Normal Mode only. Enter Master Code if operating Master Mode).\n③ Press 'I' button [Registration] from the inside of battery cover. \n④ Press 9, #, 1, # by sequence on the keypad(to Add).\n⑤ Wait until \"Ding dong dang\" melody play, then press 'I' button [Registration] from the inside of battery cover",
+		"exclusion": "① Touch Keypad with palm.\n② Enter User Code(Normal Mode only. Enter Master Code if operating Master Mode).\n③ Press 'I' button [Registration] from the inside of battery cover. \n④ Press 9, #, 3, # on the keypad by sequence (to Delete).\n⑤ Enter User Code(Normal Mode only. Enter Master Code if operating Master Mode), then press #.\n⑤ Wait until \"Ding dong dang\" melody play, then press 'I' button [Registration] from the inside of battery cover",
```

Comment:
There are a bunch of special characters that shouldn't be here. Check whitespace and spellcheck please.

Reply:
I think I removed all special characters and missing whitespace. Not seeing spelling issues. Left the escape characters, which I assume you want to keep? If not let me know and I'll remove the newlines and escaped quotes.

Reply:
> newlines and escaped quotes

need to stay. I meant the ①② etc.